new_platforms/opteesim/enc/tobject-sim.c

```diff
@@ -1,24 +1,55 @@
 /* Copyright (c) Microsoft Corporation.  All Rights Reserved. */
+#include <stdio.h>
+#include <string.h>
 #include <windows.h>
 #include <assert.h>
 #include <tee_api.h>
 
+static const char* TEEP_ObjectIDToString(
+    _In_reads_bytes_(objectIDLen) void* objectID,
+    _In_ size_t objectIDLen)
+{
+    char* buffer; 
```

Comment:
indentation looks off through this code.  Maybe a mix of tabs and spaces?

Reply:
Indeed, fixed.